pages/checkout/pay.vue

```diff
@@ -0,0 +1,85 @@
+<script lang="ts" setup>
+import { storeToRefs } from 'pinia';
+import { useMainStore } from '~/store';
+
+const { t } = useI18n();
+
+const route = useRoute();
+const store = useMainStore();
+const { plans } = storeToRefs(store);
+
+const steps = [
```

Comment:
nitpick: wouldn't it make more sense to turn this into a computed, if I understand it correctly `t` is supposed to be reactive but that would not work if evaluated during `setup`.

Reply:
we can make it a computed property, but would not be needed, every value here is static, no need for reactivity.

Reply:
> we can make it a computed property, but would not be needed, every value here is static, no need for reactivity.

Unless we add a new language and a dynamic language switch that is true.

Reply:
made it computed.